codec_test.go

```diff
@@ -132,10 +133,10 @@ var configurationDoneResponseStruct = ConfigurationDoneResponse{Response: *newRe
 
 // -------- Launch
 
-var launchRequestString = `{"seq":3,"type":"request","command":"launch","arguments":{"noDebug": true,"name":"Launch"}}`
+var launchRequestString = `{"seq":3,"type":"request","command":"launch","arguments":{"name":"Launch","noDebug":true}}` // map keys are sorted.
```

Comment:
What do you mean by "sorted"?

Reply:
`name` comes before `noDebug`. go's encoding/json package encodes map keys in sorted order -
now we use json.RawMessage, the arguments should be exactly how the json data is encoded.

Reply:
Do I understand correctly that this is a side effect of using Marshal to produce RawMessage in the expected struct? Technically the input to the decoding can be in any order, so without more detailed comments, this is a bit confusing. That said, now that I think about this - do you need maps and toRawMessage helper at all here? can you just use json.RawMessage("{...}")} to have whatever order you need?

Reply:
Done. 

Right, I think this test shouldn't care about what is in the Arguments field any more but just tests whether []byte in "arguments" are recovered to the original []byte - since Arguments is now a []byte (json.RawMessage) and how to interpret them is a business for the user of this package. Even `"arguments": "foobar"` should be ok too.

Reply:
Yes, I agree that what's inside bytes doesn't matter anymore.